fapolicy_analyzer/ui/subject_list.py

```diff
@@ -241,6 +241,14 @@ def on_view_button_press_event(self, widget, event):
         model, pathlist = treeView.get_selection().get_selected_rows()
         n_paths = len(pathlist)
         if event.type == Gdk.EventType.BUTTON_PRESS and event.button == 3:
```

Comment:
I know you didn't change this, but the nesting is getting nasty in this method. Can you move this condition to the top of the method and change to this
```
if event.type != Gdk.EventType.BUTTON_PRESS or event.button != 3:
            return False
```
Then fix up the indenting.